From: Michael Franzl <office@michaelfranzl.com>
Date: Wed, 12 Dec 2012 07:10:00 -0500
Subject: Remove rubygems from examples

This patch modifies the examples so that they
do not use rubygems, and don't modify the LOAD_PATH. The
library will be installed in a place where ruby can find it,
and the relative path ../lib is not valid anymore.
---
 examples/eventmachine.rb | 2 --
 examples/threaded.rb     | 1 -
 2 files changed, 3 deletions(-)

diff --git a/examples/eventmachine.rb b/examples/eventmachine.rb
index dd8419c..3f047a4 100644
--- a/examples/eventmachine.rb
+++ b/examples/eventmachine.rb
@@ -1,6 +1,4 @@
-$LOAD_PATH.unshift 'lib'
 
-require 'rubygems'
 require 'eventmachine'
 require 'mysql2/em'
 
diff --git a/examples/threaded.rb b/examples/threaded.rb
index 440a008..3c6ccdb 100644
--- a/examples/threaded.rb
+++ b/examples/threaded.rb
@@ -1,4 +1,3 @@
-$LOAD_PATH.unshift 'lib'
 require 'mysql2'
 require 'timeout'
 
